### DISCUSSION TITLE
Label at organization level > Push new labels to existing repos

### DISCUSSION THREAD

**Discussion by adriens** [community/community#15300] [Projects and Issues]
# ❔ Context

Actually when we create a label at the organization level, it is only available for newly created repos...

👉 but is not available of repos existing before the label creation... hence forcing us to do it manually on all the already existing.

# 🙏 Feature request

Even if it is documented on GH cf : 

![image](https://user-images.githubusercontent.com/5235127/164364880-615eebb2-6fc7-4fbf-bc5e-79a5415bf44d.png)

☝️  Would it be possible to make newly created label at org level available for all repos ❔

**Comment by adriens** [comment]
For now, we have a workaround thanks to https://github.com/cli/cli/pull/5452 cf [GitHub CLI 2.9.0](https://github.com/cli/cli/releases/tag/v2.9.0)

**Reply by Gnangom** [reply]
This site is an working place
How it works

**Comment by adriens** [comment]
Workaround with `cli/cli` commands : https://dev.to/adriens/github-cli-v290-is-out-clone-labels-feature-5452-is-here-3odk

**Comment by philmod-h** [comment]
@adriens How do you clone the labels from the organization to a repository?

**Reply by adriens** [reply]
If the repo does not yet exist just create labels at the org level then create the repo so it will inherit them

**Reply by philmod-h** [reply]
In my case, the repository already exists, I added the label to the org, how could I sync these labels from the org to the existing repo?

**Reply by adriens** [reply]
If if it's just for one repo, I would consider doing it manually or through gh cli commands. Then add labels to your org so you will inherit them in the future. If you have many repos i nwhich to add labels, I would probably write a dummy script around gh cli command to do the job on each repo
